src/components/TodoContainer.js

```diff
@@ -0,0 +1,43 @@
+import React from "react"
+import TodosList from "./TodosList";
+import Header from "./Header"
+class TodoContainer extends React.Component {
+  state = {
+    todos: [
+      {
+        id: 1,
+        title: "Setup development environment",
+        completed: true
+      },
+      {
+        id: 2,
+        title: "Develop website and add content",
+        completed: false
+      },
+      {
+        id: 3,
+        title: "Deploy to live server",
+        completed: false
+      }
+    ]
+  };
+  handleChange = id => {
+  this.setState(prevState => ({
+  todos: prevState.todos.map(todo => {
+    if (todo.id === id) {
+      todo.completed = !todo.completed
+    }
+    return todo
+  }),
+}))
+  };
+  render() {
+    return (
+      <div>
+        <Header />
+        <TodosList todos={this.state.todos} handleChangeProps={this.handleChange} />
+      </div>
+    );
+  }
+}
+export default TodoContainer
```

Comment:
Hello, it would be a great idea to end this line with a semi-colon. Kindly have a look at list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) for a refresher.